### PR TITLE
 Prevent running Redis test suite on forks

### PR DESCRIPTION
**Pull request by tezc**
- Prevent running Redis test suite on forks
- Added manual dispatch options

### CODE REVIEW

File: .github/workflows/redis-suite.yml [hunk 1]
```diff
@@ -4,21 +4,51 @@ on:
...
+    - name: prep
+      if: github.event_name == 'workflow_dispatch'
+      run: |
+        echo "REDIS_REPOSITORY=${{github.event.inputs.redis_repo}}" >> $GITHUB_ENV
+        echo "REDIS_BRANCH=${{github.event.inputs.redis_branch}}" >> $GITHUB_ENV
+        echo "REDISRAFT_REPOSITORY=${{github.event.inputs.redisraft_repo}}" >> $GITHUB_ENV
+        echo "REDISRAFT_BRANCH=${{github.event.inputs.redisraft_branch}}" >> $GITHUB_ENV
     - uses: actions/checkout@v2
+      with:
+        repository: ${{ env.REDISRAFT_REPOSITORY }}
```
Comment (yossigo) [inline .github/workflows/redis-suite.yml:41]: @tezc Did you test this as a scheduled workflow? I'd expect these variables have to be defined (to hard coded value) in that case, but maybe I'm missing something.

Comment (tezc) [reply .github/workflows/redis-suite.yml]: @yossigo You're right, I didn't try with scheduled workflow. It should be fixed right now.

**Review by yossigo — COMMENTED** [review]
@tezc I actually think the first version was good, having the ability to configure everything.

I believe the only thing that was missing was an `env:` section to set up the environment variables to the defaults even when the trigger is not workflow_dispatch. This is because inputs on workflow dispatch are simply not available on scheduled jobs.

**Comment by tezc** [comment]
@yossigo Yeah, needed default values. Then, I thought maybe we won't need to configure Redis branch and commit any time soon, so, switched back to this. Looks like you liked that one better, I'll update the PR back to first version.

**Comment by tezc** [comment]
@yossigo Updated the PR, made it configurable back again.

**Review by yossigo — APPROVED** [review]

**Pull request merged**